lib/osf-components/addon/components/search-page/template.hbs

```diff
@@ -108,27 +134,35 @@ as |layout|>
         {{!-- paginator --}}
         {{#unless @showSidePanelToggle}}
             <div local-class='topbar'>
-                <nav
-                    data-test-topbar-object-type-nav
-                    local-class='object-type-nav'
-                >
-                    <ul>
-                        {{#each this.resourceTypeOptions as |resourceType|}}
-                            <li>
-                                <LinkTo
-                                    data-analytics-name='Object type filter={{resourceType.value}}'
-                                    data-test-topbar-object-type-link='{{resourceType.value}}'
-                                    local-class='object-type-filter-link'
-                                    @route={{@route}}
-                                    @query={{hash resourceType=resourceType.value}}
-                                    {{on 'click' (fn this.updateResourceType resourceType)}}
-                                >
-                                    {{resourceType.display}}
-                                </LinkTo>
-                            </li>
-                        {{/each}}
-                    </ul>
-                </nav>
+                {{#if this.showResourceTypeFilter}}
+                    <nav
+                        data-test-topbar-object-type-nav
+                        local-class='object-type-nav'
+                    >
+                        <ul>
+                            {{#each this.resourceTypeOptions as |resourceType|}}
+                                <li>
+                                    <LinkTo
```

Comment:
Not using a button here?

Reply:
Using a `LinkTo` here as we use the `.active` class to style the resource-type currently applied (not to be mistaken with the `:active` pseudo-class in `button.scss` for buttons being clicked by users). You may ask in follow-up, why not use `<OsfLink>` then? `<OsfLink>` doesn't do well with adding the `.active` class if we are looking for active query-params.

Reply:
Cool